imports/ui/components/profile/profile_page.js

```diff
@@ -63,3 +73,20 @@ Template.profilePage.helpers({
     return Meteor.userId() && profile.userId === Meteor.userId();
   }
 });
+
+Template.profilePage.events({
+  'click #ban-user-from-profile': (e) => {
+    var username = FlowRouter.getParam("username");
+    var user = Meteor.users.findOne({username: username});
+    var profile = Profiles.findOne({ userId: user._id });
+    Profiles.update(profile._id, {$set: {banned: true}});
+    Bert.alert("User @" + username + " banned.", "default");
```

Comment:
Thank you very much. And yes, better we build things to be secure from the get go as much as we can.